src/converters/basic_sensors/moving.ts

```
@@ -0,0 +1,30 @@
+      otherExposes,
+      MovingSensorHandler.generateIdentifier,
+      'Motion Sensor (moving)',
+      (n, t) => new hap.Service.MotionSensor(n, (MovingSensorHandler.exposesName + ' ' + (t ?? '')).trim()),
+      hap.Characteristic.MotionDetected,
+      true,
+      false
+    );
+  }
+
+  static generateIdentifier(endpoint: string | undefined) {
+    let identifier = MovingSensorHandler.exposesName + '_' + hap.Service.MotionSensor.UUID;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)